website/content/v1.7/talos-guides/install/single-board-computers/rpi_generic.md

```diff
@@ -24,7 +25,7 @@ You will need
 Download the latest `talosctl`.
 
 ```bash
-curl -Lo /usr/local/bin/talosctl https://github.com/siderolabs/talos/releases/download/{{< release >}}/talosctl-$(uname -s | tr "[:upper:]" "[:lower:]")-amd64
+curl -Lo /usr/local/bin/talosctl https://github.com/siderolabs/talos/releases/download/{{< release >}}/talosctl-$(uname -s | tr "[:upper:]" "[:lower:]")-arm64
```

Comment:
I don't think this is right way to do it though, that is your client, so it might run different arch than your board. 

we also have https://www.talos.dev/v1.6/introduction/quickstart/#prerequisites

Reply:
That's a good point. It appears the people who run into this problem are using a different pi to manage their pi install. Maybe another good reason to encourage people to use homebrew (automatic architecture parsing).

The only other option would be to add something like this at the end (which is ugly and prone to failure)
```bash
if [ $(uname -m) == "x86_64" ]; then echo "amd64"; else echo "arm64"; fi
```

Reply:
For now I'll just add a 2nd section that clearly states which architecture they're downloading and give both commands.
Because the code box in the site cuts off the back part of the command it's impossible to know which which you're downloading without scrolling.

Reply:
https://talos.dev/install this script should handle it all?

Reply:
TIL that was an option. I'll update to use that script.